defaults/main.yml

```diff
@@ -1,16 +1,23 @@
 ---
 
+kafka_path                        : "/opt/kafka_{{ kafka_scala_version }}-{{ kafka_version }}"
+kafka_hosts                       : "{{ groups['bi-kafka'] }}"
```

Comment:
I think it's better to leave this as an empty list "[]".
To not rely on our internal infra stuff.